native_client/deepspeech.h

```diff
@@ -1,11 +1,15 @@
 #ifndef DEEPSPEECH_H
 #define DEEPSPEECH_H
 
+#if defined _MSC_VER
```

Comment:
Isn't this going to be a problem when using `SWIG` on Windows platform ? Shouldn't we move that inside the `ifndef SWIG` branch, instead ?

And maybe add comments on the `endif` lines, now that we have more complexity, to make it easier to follow the code flow

Reply:
@lissyx  Yes this is a possible issue, I don't know that much on how SWIG creates its glue code for other langs, I tried to get the build command and test from the taskcluster but is hard to read. If you can share the command that the taskcluster uses I'll test it.

Reply:
The `makefile ` for the python bindings is not compatible with `mingw64`, we need changes there. I tried with the `makefile ` for python binding but did not work, as long I understand we link to the built .so, so I should work with the current `_MSC_VER`

Reply:
I think what @lissyx wants here, and I agree, is to move the `#ifdef _MSC_VER` inside the `#ifndef SWIG`.

Reply:
Compiled :)